services/core-web/src/components/dashboard/noticeOfWorkHomePage/NoticeOfWorkTable.js

```diff
@@ -272,24 +271,22 @@ export class NoticeOfWorkTable extends Component {
 
   render() {
     return (
-      <TableLoadingWrapper
+      <CoreTable
         condition={this.props.isLoaded}
-        tableHeaders={getTableHeaders(this.columns())}
-      >
-        <Table
-          rowClassName="fade-in"
-          align="left"
-          pagination={false}
-          columns={applySortIndicator(
-            this.columns(this.props),
-            this.props.sortField,
-            this.props.sortDir
-          )}
-          dataSource={this.transformRowData(this.props.noticeOfWorkApplications)}
-          locale={{ emptyText: <NullScreen type="no-results" /> }}
-          onChange={handleTableChange(this.props.handleSearch)}
-        />
-      </TableLoadingWrapper>
+        columns={applySortIndicator(
+          this.columns(this.props),
+          this.props.sortField,
+          this.props.sortDir
+        )}
+        dataSource={this.transformRowData(this.props.noticeOfWorkApplications)}
+        tableProps={{
+          rowClassName: "fade-in",
+          align: "left",
```

Comment:
This can be a default class on CoreTable as it goes with the transition of the loading skeleton and every table should inherit